dipy/tracking/tests/test_utils.py

```diff
@@ -703,3 +702,28 @@ def test_curvature_angle():
     with pytest.warns(UserWarning):
         npt.assert_equal(min_radius_curvature_from_angle(0, 1),
                          min_radius_curvature_from_angle(np.pi/2, 1))
+
+
+def test_seeds_directions_pairs():
+    positions = np.random.random((10, 3))
```

Comment:
np.random.random will be soon deprecated so we avoid it.

see abov,  in this file, the use of `set_random_number_generator` + `rng` instead.